playground/house2.py

```diff
@@ -0,0 +1,122 @@
+import os
+import time
+from random import random
+from board import Board, Stone
+from referee import Referee
+from bot import Bot
+import util
+from repo.game import save_game_result
+from connect_db import DBConfig
+
```

Comment:
Please sort import orders

Reply:
for reference, my IDE recommends as follows

```
import os
import time
from random import random

import util
from board import Board
from bot import Bot
from connect_db import DBConfig
from referee import Referee
from repo.game import save_game_result
```